www/http/account_test.go

```diff
@@ -65,4 +66,43 @@ func TestAccount(t *testing.T) {
 		assert.Equal(t, w.Code, 400)
 		fmt.Println(w.Body)
 	})
+
+	t.Run("Shall return an account", func(t *testing.T) {
```

Comment:
Let's move these tests to a new function: `TestAccountByNumber`

Also, try to cover the situation that account can't be found, like number is `acc.Number()+1`